scenarios/README.md

```diff
@@ -401,6 +401,41 @@ The scenario named `custom` can be used to pass any custom filter variable like
 crank --config https://github.com/aspnet/benchmarks/blob/main/scenarios/dotnet.benchmarks.yml?raw=true --scenario custom --profile aspnet-perf-win --variable filter=*LinqBenchmarks*
 ```
 
+## HTTP client benchmarks
+
+These scenarios are running various HTTP client benchmarks.
+
+### Sample
+
+```
+crank --config https://raw.githubusercontent.com/CarnaViire/Benchmarks/http3-benchmarks/scenarios/httpclient.benchmarks.yml --scenario httpclient-kestrel-get --profile aspnet-perf-lin
+```
+*TODO: update to `https://raw.githubusercontent.com/aspnet/Benchmarks/main/scenarios/httpclient.benchmarks.yml`*
+
+### Available scenarios
+
+- `httpclient-kestrel-get`: Execute GET requests to the server. Client is HttpClient, server is Kestrel (minimal API).
+
+#### Arguments
+
+- HTTP Version to use:
+  - `--variable httpVersion="1.1"` (default)
+  - `--variable httpVersion="2.0"`
+  - `--variable httpVersion="3.0"`
+- Number of clients:
+  - `--variable numberOfClients=<N>` (default: `1`)
```

Comment:
Since this refers to the number of HttpClient instances (I think right?) can we consistently use the term HttpClient instead of Client? E.g. numberOfHttpClients, concurrencyPerHttpClient, etc. The term "client" is vague and could mean lots of things.

Reply:
This was mostly because we intend to compare to other things like LLHTTP, and it would be an LLHTTP connection in that case; thought, the term "HTTP client" might also apply to it, what do you think @MihaZupan @scalablecory?